### ISSUE TITLE
Localised trajectories

### ISSUE THREAD

**Issue by mtakanen** [gschivley/climate-life-events#12]
Cool project! As we know predicted temperature changes are not even globally. I would like to see trajectories based on local data if available. It could help those living on the higher latitudes awaken on the issue, e.g. in Finland. I can also help with coding if needed.

**Comment by gschivley** [comment]
It's a great idea, and I would definitely need help with coding. To implement we would need:

- A way for users to select their location (click on map of globe, dropdown of cities, enter city?)
- A new data source for regional temperature anomalies. Could go with [CMIP5](http://climexp.knmi.nl/selectfield_cmip5.cgi?id=someone@somewhere) ([wikipedia description](https://en.wikipedia.org/wiki/Coupled_model_intercomparison_project)). The model runs include both historic and projections, so there wouldn't be any need to use different data sets. But data are only available in netCDF format, which I've never worked with. There is at least one Python package for extracting netCDF data.
- A way to match the user selected location with regional data.
- Some level of data aggregation? The temp change data are at some fine level of resolution (a couple degree lat/lon).

I know people who can help explain the CMIP5 data. If this still sounds like a project you're interested in I'll reach out to them.

**Comment by gschivley** [comment]
Could also add other regional impacts that are more relatable than mean temperature.

- Days above XX temp (40C/100F)
- Blooming season
- Migration patterns
- ??

Not sure if any of these data are available or where to start looking for them.

**Comment by mtakanen** [comment]
Yes, more personal the better. Impacts in ecosystem are interesting but maybe not try to include all available parameters in a single app that UI wouldn't get too busy. Local sea level change could be another future enhancement. 

The tasks you've written in your first comment are very good. I've never worked with climate data so any help is welcome. Other tasks feel straightforward.

How about using HTML5 Geolocation API to locate user? It should work on all modern browsers.
https://gist.github.com/mtakanen/2a9878e288b59d5c5eb46879345eb66b

**Comment by gschivley** [comment]
My immediate thoughts:
What would this look like to the user? Just a button that gives us their location? Does it then show it on a map? What if the user wants to select somewhere else?

I'll try to figure out netCDF files and regional aggregation. The rest of this week is busy for me but I'll check in when possible.

**Comment by mtakanen** [comment]
My gist is just a POC to get user's current location. No need to show button or map when integrated in the app. Selecting somewhere else would require a map or place name selector.

Cool, the data portal looks quite cryptic to me.
